docs/getting-started/getting-started-with-docker.rst

```diff
@@ -22,15 +24,143 @@ There is a good chance that you have heard of containerization. A container is e
 
 Docker is a well-known type of containerization software. It provides a fast environment for both users and developers. A developer can package software and its dependencies into an Docker image, and then share that Docker image with users. Users who download your Docker image and run it with the Docker program will be able to run the software you packaged without having to handle the installation of your program's prerequisites or anything else; it's ready to go as-is.
```

Comment:
Above this I suggest adding a section title "Why use Docker containers? ~~~~" Then this paragraph answers it.

Reply:
Do we also want a section about "Where can I run Docker containers?". Locally, in the cloud, and then you could bring up the HPC text below, but point out that you can run them in HPCs with some extra steps...

Reply:
Thoughts? https://github.com/dockstore/dockstore-documentation/commit/803506553c522196550b6e0598875fefc0dc25ef

Reply:
looks good!